app/kubemci/pkg/gcp/backendservice/backendservicesyncer.go

```diff
@@ -58,6 +59,7 @@ func (b *BackendServiceSyncer) EnsureBackendService(lbName string, ports []ingre
 		be, beErr := b.ensureBackendService(lbName, p, hcMap[p.Port], npMap[p.Port], igLinks, forceUpdate)
 		if beErr != nil {
 			beErr = fmt.Errorf("Error %s in ensuring backend service for port %v", beErr, p)
+			glog.Errorf("Error ensuring backend service: %v. Continuing.", beErr)
```

Comment:
Use fmt.Printf() instead of glog.Errorf().
glog.Errorf() also prints the timestamp, file name and line number. Users of the binary dont care about those.

Reply:
done.